modules/using-wildcard-routes.adoc

```diff
@@ -0,0 +1,317 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring-ingress-controller
+
+[id="using-wildcard-routes_{context}"]
+= Using wildcard routes
+
+The HAProxy router has support for wildcard routes, which are enabled by setting
```

Comment:
We're trying to convert the term router to ingress controller, s/router/ingress controller/. Maybe start off with the following to help with the conversaion:

The HAProxy ingress controller, i.e. router, has support for wildcard routes...

Reply:
Thank you for clarifying - I thought this was the case, but wanted to confirm because I've heard both terms used interchangeably.